src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -1112,6 +1116,15 @@ public void AcceptNewConnection(Stream stream, Action disconnect, EndPoint remot
             }
         }
 
+        private static void ValidateCompatibility(RachisHello initialMessage)
+        {
+            if (ServerVersion.IsNightlyOrDev(initialMessage.ServerBuildVersion) == false &&
+                initialMessage.ServerBuildVersion <= 41010)
```

Comment:
Old versions will not send anything so you will get 0. What will happen then?
Also 4.2.0 will not send anything. Actually nodes <= 41010 are not sending anything so I'm not sure if there is a point of having this at the moment? Can be used in future, but not know?

Reply:
you are correct, at all points :)